aggregator/src/recursion/circuit.rs

```diff
@@ -68,21 +76,39 @@ fn accumulate<'a>(
 
 #[derive(Clone)]
 pub struct RecursionCircuit<ST> {
+    /// The verifying key for the circuit.
```

Comment:
Basicaly it is not the verifying key but the common params of the recursion circuit. The so called "vk" part which identify the circuit is passed via `preprocessed_digest` PI.